migration/schema/v1/tables.go

```diff
@@ -1011,3 +1011,39 @@ func ConvertV2SamlToIdpConfig(samlMap map[string]any) (v3Config *IdpConfig, err
 	v3Config.Enabled = true
 	return
 }
+
+// Group is respretation of a row in "platformdb"."groups" table
+type Group struct {
+	GroupID     string `json:"group_id,omitempty"`
+	DisplayName string `json:"display_name,omitempty"`
+	RealmID     string `json:"realm_id,omitempty"`
+}
+
+func ConvertToGroup(grp map[string]any) (g *Group) {
+	group := &Group{}
+	group.GroupID = grp["_id"].(string)
```

Comment:
Given we're operating on `any`/`interface{}` types, trying to blindly assert a type can lead to a panic. For example, if `grp` was an empty `map[string]any`, `grp["_id"]` is `nil`, and asserting that `nil` is a `string` will cause the program to panic, e.g. [example](https://go.dev/play/p/9qnMNVQp9hf). This is why the two-value test form is used for accessing map values with keys and asserting types on interfaces. Examples of this can be found in the ConvertV2DirectoryToV3IdpConfig function.